pkg/utils/jolokia_client/jolokia_client.go

```diff
@@ -139,12 +142,14 @@ func resolveJolokiaRequestParams(namespace string,
 		for _, oneVar := range envVars {
 			if !userDefined && oneVar.Name == "AMQ_USER" {
 				jolokiaUser = getEnvVarValue(&oneVar, &podNamespacedName, client, nil)
-			}
-			if !userDefined && oneVar.Name == "AMQ_PASSWORD" {
+			} else if !userDefined && oneVar.Name == "AMQ_PASSWORD" {
 				jolokiaPassword = getEnvVarValue(&oneVar, &podNamespacedName, client, nil)
-			}
-			if oneVar.Name == "AMQ_CONSOLE_ARGS" {
+			} else if oneVar.Name == "AMQ_CONSOLE_ARGS" {
 				jolokiaProtocol = getEnvVarValue(&oneVar, &podNamespacedName, client, nil)
+			} else if oneVar.Name == "JAVA_ARGS_APPEND" {
+				jolokiaProtocol, port = getProtocolFromJavaArgs(oneVar.Value)
+			} else if oneVar.Name == "AMQ_NAME" {
```

Comment:
same question, name from an env var, why?

Reply:
This is where the name come from. So I think make the name not hardcoded just in case people changes it.

Reply:
The port and name added here don't add any new behaviors it's just for this particular case, neither it'll break existing ones. I'll remove them because they are not part of fixing the main issue.